packages/vite/src/node/optimizer/index.ts

```diff
@@ -1232,3 +1236,26 @@ export async function optimizedDepNeedsInterop(
   }
   return depInfo?.needsInterop
 }
+
+const MAX_TEMP_FOLDER_AGE_MS = 24 * 60 * 60 * 1000
+export async function cleanupTemporalDepsCache(
```

Comment:
Me being nitpicky 😬  but I'm not sure if "temporal" is the right word here. Maybe "stale"?